### PR TITLE
Add support for Nice Flor-s remote

### PR DESCRIPTION
**Pull request by samueltardieu**

### CODE REVIEW

**Comment by samueltardieu** [comment]
Captures are available from https://rfc1149.net/tmp/nice-flor-s-captures.zip (first one uses first button of the remote, others use the second button).

File: src/devices/nice_flor_s.c [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/** @file
+    Nice Flor-s
```
Comment (zuckschwerdt) [inline src/devices/nice_flor_s.c:2]: `Nice Flor-s remote for car gates.` perhaps? not the dot, also below.

File: src/devices/nice_flor_s.c [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+/**
+Nice Flor-s
+
+Protocol description:
+The protocol has been analyzed at this link: http://phreakerclub.com/1615
+*/
```
Comment (zuckschwerdt) [inline src/devices/nice_flor_s.c:17]: Can you add a summary? At least the data layout.

File: src/devices/nice_flor_s.c [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+#include "decoder.h"
+
+static int nice_flor_s_callback(r_device *decoder, bitbuffer_t *bitbuffer)
```
Comment (zuckschwerdt) [inline src/devices/nice_flor_s.c:21]: `nice_flor_s_decode` is preferred

File: src/devices/nice_flor_s.c [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    int repeat = (b[0] & 0xf) ^ 0xf ^ button_id;
+    char serial[8], code[5];
+    // Encrypted serial is made of nibbles 2 and 7 to 12
+    snprintf(serial, sizeof serial,
+             "%x%x%02x%02x%x",
+             b[1] >> 4, b[3] & 0xf, b[4], b[5], b[6] >> 4);
+    // Encrypted rolling code is made of nibbles 3 to 6
+    snprintf(code, sizeof code,
+             "%x%02x%x",
+             b[1] & 0xf, b[2], b[3] >> 4);
```
Comment (zuckschwerdt) [inline src/devices/nice_flor_s.c:46]: can you use `int code =  ((b[1] & 0xf) << 12) | (b[2] << 4) | (b[3] >> 4);` and `DATA_FORMAT, "%04x", DATA_INT, code` below, also for serial?

File: src/devices/nice_flor_s.c [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+        .name           = "Nice Flor-s",
+        .modulation     = OOK_PULSE_PWM,
+        .short_width    = 500 ,   // short pulse is ~500 us + ~1000 us gap
+        .long_width     = 1000,   // long pulse is ~1000 us + ~500 us gap
+        .sync_width     = 1500,   // sync pulse is ~1500 us + ~1500 us gap
+        .gap_limit      = 2000,
+        .reset_limit    = 5000,
+        .tolerance      = 100,
+        .decode_fn      = &nice_flor_s_callback,
+        .disabled       = 0,
```
Comment (zuckschwerdt) [inline src/devices/nice_flor_s.c:81]: as there is not MIC make this `disabled = 1`

File: src/devices/nice_flor_s.c (hunk 1 above)
Comment (samueltardieu) [reply src/devices/nice_flor_s.c:2]: "for gates" (people use it for portal gates also). Also added in README.

File: src/devices/nice_flor_s.c (hunk 5 above)
Comment (samueltardieu) [reply src/devices/nice_flor_s.c]: Also disabled in README

**Pull request merged**